### DISCUSSION TITLE
My account restricted, what can I do with it?

### DISCUSSION THREAD

**Discussion by KKeylin** [codewars/codewars.com#3145] [Q&A] [answered]
Hello

I haven't visited CodeWars for a few years, and now I'm back, and see the wrong statistics, and notifications, that my account is restricted. 

https://www.codewars.com/users/KKeylin

I used this service honestly, did not increase my rating, and did not post other people's solutions. At this moment, I returned to start learning a new programming language with practice, and I would like to return my account to its normal state. Alas, I haven’t received any emails these years because I lost access to my hosting where my email was, so I couldn’t receive any notifications.

Please let me know what I need to do to normalize my account and the statistics on it.

**Comment by Kacarott** [comment]
I was not the one who restricted your account, but are you really trying to tell me that you managed to come up with [the exact same solution](https://www.codewars.com/kata/reviews/5cd9a996efd18d0001cd488e/groups/5da8e40a42d79100012edca8) as 150+ other people, on a 3kyu kata?

From what I can tell, the other solutions besides this one look alright, but being honest with us is a good first step to potentially getting your account back.

**Reply by KKeylin** [reply]
I’ll tell you how it is - I haven’t been on the platform for quite a long time, but I remember for sure that I have never used this platform for anything other than real training, and learning from solutions to the problems I solved. I just went in, decided how it would turn out, and then looked at ready-made solutions to learn the best practices and some tricks in JS.

Sometimes I went through some simple katas when I showed my friends the platform and showed how problems were solved here using some examples, in which these friends and acquaintances of mine could also begin to train and find solutions. 

On the other hand, my spasmodic growth may also be related to my spasmodic visits to the platform - one way or another, I grew as a programmer, and I sometimes moved on to more complex tasks because I could complete them.

In addition to this kata, I have 2 more katas that I passed, and which are for 4 kyu, and this is not much different in difficulty from 3 kyu, and to be honest, I don’t even remember how and when I solved them, but I have them in decisions.

Looks suspicious?.. Perhaps, but as you say, other decisions do not cause suspicion, and I literally don’t remember being involved in this kind of thing. For me, cheating is what killed interest in everything in principle, and therefore I doubt that I would do such a thing. I don’t know, maybe someone was interested in using something like ArtMoney in order to get maximum experience or gold in Diablo 2, but for me, it immediately discouraged me from further playing, and in real life, such tools discourage me do something. As far as I remember, these numbers meant something to me, and therefore I can assume that I did not cheat, but I’m telling it like it is - it was so long ago that I don’t remember such an episode at all.

**Reply by KKeylin** [reply]
And again, you can see that I have worked and continue to work in the field related to the application of JS on a daily basis for the last 6 years, information about this is on my LinkedIn profile, because this is not a result that came out of a vacuum, and this is how I I suddenly found myself with a black belt in a language that I had never practiced.

**Reply by KKeylin** [reply]
And because of the limitations, I can’t even see my solution, so I don’t even see it, try to understand “Can I write this solution?”

In general, if this kata is a reason for restriction, but the situation does not look clear, I would suggest simply removing it from my profile so that there is nothing suspicious there, but I remain of the opinion that I am unlikely to cheat - it’s just not the same what I prefer to do and what makes sense.

On the other hand, this decision is not an admission of guilt, but the fact that I understand that it is useful for the platform to have protection against deception and plagiarism, and sometimes such tools have false positives, but the filter should work, as this makes the platform better.

**Reply by Kacarott** [reply, marked as answer]
I have lifted the restrictions from your account, but I have not restored your points. We will wait for the mod who placed the restrictions to decide what to do about that. But now you can see that the kata I linked was definitely a copy, there is no doubt about it.

**Reply by KKeylin** [reply]
Thank you
Yes, I looked it up, it looks like a copy-paste of someone else’s solution, I don’t write like that and most likely it was plagiarism.

I apologize for this incident, I really don’t remember how and why this answer appeared on my profile, but I won’t play conspiracy theories, and I’ll just say what I think. It’s clear that something that looks like shit and smells like shit most likely is it, I just really don’t understand why it ended up on my profile, but in any case, this is a useful experience for me, and this situation will not happen again.

**Reply by Blind4Basics** [reply]
Hello,

Looking at your profile, I actually don't know why the restriction went so far. I guess it got caught in a batch, handling your account amongst many others, and I didn't look hard enough (the few suspicious kata where showing up right at the top of the page and it looks like I didn't go look lower in the page). Sorry about that. I'll revalidate what looks valid (edit: done).

Cheers

**Comment by Dalerjanskiy** [comment]
HELLO
